Consolidation OOP/consolidate/mesher/meshes.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pdb
```

Comment:
pdb should not be imported as the used will not need it. When using pdb in your code, I recommend to import it and use it in the same line this way:
`import pdb; pdb.set_trace()`
So when you delete the trace, you will not forget to delete the import.